### PR TITLE
Aankirz: Fixes issue 2975 interop_invalid_year

### PR DESCRIPTION
**Pull request by Aankirz**
## Description
Fixes [issue 2975](https://github.com/web-platform-tests/wpt.fyi/issues/2975)
- To improve the user experience
- Whenever user enters any invalid year (Interop) , redirects to a sorry page.

## Review Information
- Try entering any invalid year to https://wpt.fyi/interop-XXXX

## Changes
- Added a new file `interop-404.js` in `webapp/components`
- Modified `interop_handler.go`

### CODE REVIEW

**Comment by Aankirz** [comment]
Here's the local build as well:

https://user-images.githubusercontent.com/95562947/229319637-1201375a-9e0f-4180-93bd-4269b4ca5ea5.mp4

File: webapp/components/interop-404.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <a href="/">
+      <paper-button>Go to the home page</paper-button>
+    </a>
+`;
+  }
+
+  static get is() { return 'interop-404'; }
+}
+
+customElements.define(Interop404.is, Interop404);
```
Comment (DanielRyanSmith) [inline webapp/components/interop-404.js:37]: New line here

File: webapp/interop_handler.go [hunk 2]
```diff
@@ -62,3 +62,8 @@ func interopHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	RenderTemplate(w, r, "interop.html", data)
 }
+
+func interopError(w http.ResponseWriter, r *http.Request){
+	RenderTemplate(w, r, "interop_error.html", nil)
+
+}
```
Comment (DanielRyanSmith) [inline webapp/interop_handler.go:69]: New line here

**Review by DanielRyanSmith — APPROVED** [review]
Sorry for the delay!

This looks good, but it looks like there are some lint issues that need to be taken care of - specifically related to Windows line endings. Googling "Replace CRLF with LF" might find some answers to how to handle this, but I have not had to fix this myself. This should be fine to merge after fixing the lint check.

File: webapp/components/interop-404.js (hunk 1 above)
Comment (Aankirz) [reply webapp/components/interop-404.js:37]: Oo sorry for that 😅

**Comment by KyleJu** [comment]
This PR seems abandoned. Closing it for now and feel free to open it back

**Pull request closed without merging**